src/commands/createNewProject/createNewProject.ts

```diff
@@ -69,6 +69,10 @@ export async function createNewProject(
         await projectCreator.addNonVSCodeFiles();
 
         await initProjectForVSCode(actionContext, functionAppPath, language, runtime, projectCreator);
+        // Functions has a dependency on C# that creates build artifacts: https://github.com/Microsoft/vscode-azurefunctions/issues/658
+        if (language !== ProjectLanguage.CSharp) {
```

Comment:
I try not to have language-specific code in this file. Please move this into a language-specific projectCreator.

Also double check the languages when you move it into a project creator. This really only needs to happen for languages that explicitly have the 'func extensions install' task. For example - Java doesn't use that task and thus doesn't need this.